tests/functional/aws-node-sdk/test/serverAccessLogs/testServerAccessLogFile.js

```diff
@@ -0,0 +1,2645 @@
+                        await s3.getObject({ Bucket: 'xxx', Key: 'key' }).promise();
+                    } catch {
+                        return;
+                    }
+                };
+                return {
+                    method,
+                    methodName: 'getObjectError',
+                    expected: [
+                        {
+                            ...commonProperties,
+                            operation: 'REST.GET.OBJECT',
+                            action: 'GetObject',
+                            httpCode: 404,
+                            errorCode: 'NoSuchBucket',
+                            bucketOwner: null,
+                            bucketName: 'xxx',
+                            httpMethod: 'GET',
+                            objectKey: 'key',
+                        }
+                    ],
+                };
+            })()
+            // TODO: CLDSRV-799
+            // (() => {
+            //     // Test errorCode is set.
+            //     const method = async () => {
+            //         try {
+            //             await s3.deleteBucket({ Bucket: 'UPPERCASE'}).promise();
+            //         } catch {
+            //             return;
+            //         }
+            //     };
+            //     return {
+            //         method,
+            //         methodName: 'bucketDeleteError',
+            //         expected: [
+            //             {
+            //                 ...commonProperties,
+            //                 operation: 'REST.DELETE.BUCKET',
+            //                 action: 'DeleteBucket',
+            //                 httpCode: 404,
+            //                 errorCode: 'NoSuchBucket',
+            //                 bucketOwner: null,
+            //                 bucketName: 'xxx',
+            //                 httpMethod: 'DELETE',
+            //             }
+            //         ],
+            //     };
+            // })()
+        ];
+
+        before(async function () {
+            if (config.serverAccessLogs.mode === serverAccessLogsModes.DISABLED) {
+                this.skip();
+            }
+            if (process.env.AWS_ON_AIR) {
+                this.skip();
+            }
+            if (!fs.existsSync(path.dirname(logFilePath))) {
+                throw new Error('Logs directory does not exist');
+            }
+        });
+
+        after(async () => {
+            truncateLogFileIfExists(logFilePath);
+        });
+
+        beforeEach(async () => {
+            truncateLogFileIfExists(logFilePath);
+        });
+
+        afterEach(async () => {
+            const lastAction = await cleanupBuckets(s3, bucketName);
+            await waitForAction(logFilePath, lastAction);
+            truncateLogFileIfExists(logFilePath);
+        });
+
+        for (const operation of operations) {
+            it(`check ${operation.methodName} log entry`, async () => {
```

Comment:
Worth being more descriptive here

```suggestion
            it(`should log correct ${operation.methodName} operation with all required fields`, async () => {
```

Reply:
done